.github/workflows/functional_tests.yml

```diff
@@ -0,0 +1,26 @@
+name: Run functional tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Run diKTat from cli
+        run: |
+          curl -sSLO https://github.com/pinterest/ktlint/releases/download/0.37.1/ktlint && chmod a+x ktlint
```

Comment:
It would be nice to get versions of ktlint and diktat from pom.xml. This way we won't need to update them manually in workflow.